VideoLocker/res/values/styles.xml

```diff
@@ -280,7 +280,7 @@
         <item name="android:layout_height">@dimen/icon_view_standard_width_height</item>
     </style>
 
-    <style name="discussion_thread_icon">
+    <style name="discussion_thread_icon" tools:targetApi="17">
```

Comment:
nit-pick: could we put this only on the `<item>` that needs it? (in this case, just `<item name="android:layout_marginEnd">`)